test/org/sagebionetworks/bridge/dynamodb/DynamoUploadSchemaDaoTest.java

```diff
@@ -677,7 +677,38 @@ public void getSchemaByIdAndRevNotFound() {
 
     @Test
     public void getUploadSchemasForStudy() {
-        // mock result
+        // mock result. Create three upload schemas with the same schemaId but different revisions. 
+        // Only the highest revision should be returned (rev 3).
+        DynamoUploadSchema schema1 = new DynamoUploadSchema();
+        schema1.setSchemaId("AAA");
+        schema1.setRevision(1);
+        DynamoUploadSchema schema2 = new DynamoUploadSchema();
+        schema2.setSchemaId("AAA");
+        schema2.setRevision(2);
+        DynamoUploadSchema schema3 = new DynamoUploadSchema();
+        schema3.setSchemaId("AAA");
+        schema3.setRevision(3);
+        List<UploadSchema> mockResult = ImmutableList.<UploadSchema>of(schema1, schema2, schema3);
```

Comment:
Instead of 3 revisions of the same schema, this test would be more useful if it were 2 revisions x 2 schemas.